src/mame/drivers/astrcorp.cpp

```diff
@@ -552,7 +527,7 @@ void astrocorp_state::machine_start()
 	m_lamps.resolve();
 }
 
-
+#if 0
```

Comment:
Please just nuke out this whole preprocessor block (and move the TODO note to where it belongs, i.e. `skilldrp` machine_config fn)